.pre-commit-config.yaml

```diff
@@ -5,13 +5,13 @@ repos:
     hooks:
     -   id: black
         args: [--safe, --quiet]
-        language_version: python3.6
+        language_version: python3
```

Comment:
question - if https://github.com/miki725/black/blob/f3e287a29ae6b209ab489ed66dac42f03137df7b/.pre-commit-hooks.yaml declares a language versions, why do we do the same?

Reply:
they haven't tagged a release with the changed value

Reply:
i see, then lets have a followup issue to sort out once they do ^^